src/ui/server/api/keystone/get-ordnance-survey-address/index.ts

```diff
@@ -0,0 +1,41 @@
+import { ApolloResponse } from '../../../../types';
+import apollo from '../../../graphql/apollo';
+import ordnanceSurveyAddressQuery from '../../../graphql/queries/ordnance-survey-address';
+
+/**
+ * keystone GraphQL API call for ordnance survey address
+ * returns address or errors
+ * @param {String} postcode
+ * @param {String} houseNumber
+ */
+const getOrdnanceSurveyAddress = async (postcode: string, houseNumber: string) => {
+  try {
+    const queryParams = {
+      postcode,
+      houseNumber,
+    };
+
+    const query = ordnanceSurveyAddressQuery;
```

Comment:
Could just use `ordnanceSurveyAddressQuery` instead of defining new query variable? Recently changed this in other queries to avoid additional lines